### ISSUE TITLE
In korean Language game client price graph is not working.

### ISSUE THREAD

**Issue by neo6633** [PoE-Overlay-Community/PoE-Overlay-Community-Fork#247]
# 🐞 bug report

**Describe the bug**

In korean Language game client price graph is not working.

**To Reproduce**

It just says "오류가 발생했습니다." on there position.

**Expected behavior**

**Screenshots**

https://i.imgur.com/FX66ADT.png

## 🌍  Your Environment
    OS: Win10 x64
    Version: 0.8.5
    PoE: Kakao client Korean

**Comment by WhiteFang5** [comment]
Unfortunately I don't speak Korean, so can you translate what's written there?

**Comment by neo6633** [comment]
That means "An error has occurred."

**Comment by MegaEzik** [comment]
A similar problem on the client with the Russian language.
https://imgur.com/a/TvPiC44

**Comment by baterflyrity** [comment]
> 
> 
> A similar problem on the client with the Russian language. https://imgur.com/a/TvPiC44

+1 No plot is shown in russian.

**Comment by WhiteFang5** [comment]
This issue occurs because the external sites used to obtain the graph or price prediction don't support Korean or Russian as input language. We simply send your copy-pasted item to them, and they can't parse said item (because it's in Korean/Russian) and return an error.

**Comment by baterflyrity** [comment]
> 
> 
> This issue occurs because the external sites used to obtain the graph or price prediction don't support Korean or Russian as input language. We simply send your copy-pasted item to them, and they can't parse said item (because it's in Korean/Russian) and return an error.

So do you mean "wont fix" or "f*k off" when close actual issue?

**Comment by WhiteFang5** [comment]
> > This issue occurs because the external sites used to obtain the graph or price prediction don't support Korean or Russian as input language. We simply send your copy-pasted item to them, and they can't parse said item (because it's in Korean/Russian) and return an error.
> 
> So do you mean "wont fix" or "f*k off" when close actual issue?

Yeah, we won't fix it, because we actually can't. It's outside of our app. When poeprices supports multiple languages for item parsing, we can re-visit this and implement it on our side too. So meanwhile there isn't much we can do about it.

**Comment by WhiteFang5** [comment]
@MegaEzik ah, I see now. I checked the Russian screenshots again, and that appears to be a completely different issue than the original bug report. I've separated it and created a new issue for it (https://github.com/PoE-Overlay-Community/PoE-Overlay-Community-Fork/issues/267). My original reply & closing of the ticket is still valid for the originally posted bug report of "An error has occurred." being shown in Korean.
